packages/orchestration/scripts/fetch-chain-info.ts

```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/env tsx
```

Comment:
files in `scripts` should be executable. Can you get that ts-blank-space incantation in `codegen` to work with `/usr/bin/env`?